mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -88,6 +104,7 @@ fi
 export HZN_LISTEN_IP=${HZN_LISTEN_IP:-127.0.0.1}   # the host IP address the hub services should listen on. Can be set to 0.0.0.0 to mean all interfaces, including the public IP, altho this is not recommended, since the services use http.
 export HZN_TRANSPORT=${HZN_TRANSPORT:-http}
 
+export EXCHANGE_IMAGE_NAME=${EXCHANGE_IMAGE_NAME:-openhorizon/${ARCH}_exchange-api}
```

Comment:
Where are these images coming from for ppc64? The primary design point of `deploy-mgmt-hub.sh` is that with a single command it just works. How will that be the case for ppc64?

Reply:
Sure full set of images and RPM packages for `pcc64le` should be in the `Docker Hub` corresponding repos under `openhorizon` account. In this case `deploy-mgmt-hub.sh` should work as designed. I think it is a responsibility somebody in Open Horizon team.
To test changes I used custom-made images by my colleagues and myself and store ones in Artifactory repo. I used a special script `env.sh` to define/rewrite environment variables settings used in the `deploy-mgmt-hub.sh` and run this script before `deploy-mgmt-hub.sh` as `. env.sh`.

Reply:
Leave `ppc64le` mention in README but added mention about experimental in it and usage function too.

Reply:
You didn't follow my instructions. There is no reason to mention ppc64le support in the readme, because currently it won't work for anyone but your team (because no one else has the images, and building them for every mgmt hub component is non-trivial). The readme should contain information that is needed by everyone. Remove the mention of ppc64le from the readme!

Reply:
@bmpotter ppc64le mentioning was removed from readme